README.md

```diff
@@ -72,9 +72,9 @@ docker run -d \
   ghcr.io/westerveltco/django-email-relay:latest
 ```
 
-It is recommended to pin to a specific version, though if you prefer you can ride the lightning by always pulling the `latest` image.
+It is recommended to pin to a specific version, though if you prefer, you can ride the lightning by always pulling the `latest` image.
 
-See the documentation [here](#configuration) for general information about configuring `django-email-relay`, [here](#relay-service-1) for general information about configuring the relay service, and [here](#docker-1) for information specifically related to configuring the relay service as a Docker container.
+See the documentation [here](#configuration) for general information about configuring `django-email-relay`, [here](#relay-service-1) for available information about configuring the relay service, and [here](#docker-1) for information specifically related to configuring the relay service as a Docker container.
```

Comment:
Not sure about this one. Could just shorten both uses of "general information" to "information"

Reply:
```suggestion
See the documentation [here](#configuration) for general information about configuring `django-email-relay`, [here](#relay-service-1) for information about configuring the relay service, and [here](#docker-1) for information specifically related to configuring the relay service as a Docker container.
```